Sources/ArcGISToolkit/Components/OfflineMapAreasView/OfflineMapAreasView.MapViewModel.swift

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import Combine
+import Foundation
+import SwiftUI
+
+extension OfflineMapAreasView {
+    /// The model class for the offline map areas view.
+    @MainActor
+    class MapViewModel: ObservableObject {
+        /// The portal item ID of the web map.
+        private let portalItemID: String?
```

Comment:
I think it would be a good idea to not use String here, but continue to use the `Item.ID` for these type of properties.